vmm-reference-simple-handler.md

```diff
@@ -0,0 +1,137 @@
+# [Simple Handler](./../src/devices/src/virtio/net/simple_handler.rs)
+
+`SimpleHandler` contains methods for handling Tx/Rx queues. This handler is generalized over the implementaton of queues (does not make any assumptions the way queues are implemented). 
```

Comment:
`SimpleHandler` contains methods for handling Tx and Rx queues. This handler is generalized over the implementation of queues (does not make any assumptions about the way queues are implemented).